fetch-key-vault-secrets/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Fetch Key Vault Secrets
+description: Action to Fetch Secrets stored in Azure Key Vault
+inputs:
+  KEY_VAULT:
+    description: Name of the Key Vault the secrets are being stored
+    required: true
+  SECRETS:
+    description: Comma-separated list of secret names to fetch values from
+    required: true
```

Comment:
It might be good to name these the same as the deprecated action to make migrating to this really easy: https://github.com/Azure/get-keyvault-secrets

(Also the convention for GitHub Actions seems to be lowercase with dashes instead of underscores.)